wordpress-mobile/WordPress-Android
# Update gutenberg mobile version to use S3 maven repo

This PR updates `gutenbergMobileVersion` to the version published to S3 by https://github.com/wordpress-mobile/gutenberg-mobile/pull/3325. This change is due to [Bintray shutting down their services](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/).

**Relevant PRs:**
https://github.com/WordPress/gutenberg/pull/30421
https://github.com/wordpress-mobile/gutenberg-mobile/pull/3325

**To test:**
* Make sure `localGutenbergMobilePath` is commented out in `local-builds.gradle` to make sure you are using the binary dependency for gutenberg
* Verify gutenberg editor works as expected (Either all blocks will work or none of it will, so you don't need to verify every block)

**PR submission checklist:**
